common:
  out_path_: "/home/fyx/ufo_hiahia/src/out/"
  kNumOmpCores_: 8   # cores
  save_: true
  mapping_init_: false
  is_pcd_: false

session:
  data_path_: "/home/fyx/ufo_hiahia/src/in/bins/"
  label_path_: "/home/fyx/ufo_hiahia/src/in/labels/"
  pose_path_: "/home/fyx/ufo_hiahia/src/in/pose.txt"
  init_: 5   # the num of frame to initialize
  start_: 5  # the start index in dataset, please check it
  end_: 200

ssc:
  calib_path_: "calib/"
  seg_path_: "seg/"
  pcd_path_: "pcd/"

  sensor_height_: 2.4

  min_dis_:  1.8  # /m   use 0.0 for parkinglot dataset
  max_dis_: 40.0  # ratio

  min_angle_: 0.0  # /deg
  max_angle_: 360.0

  min_azimuth_: -30.0  # /deg
  max_azimuth_: 60.0

  range_res_: 0.4
  sector_res_: 1.2
  azimuth_res_: 2.0

  max_intensity_: 255.0  # intenisty calibration
  correct_ratio_: 0.5
  correct_radius_: 0.3
  search_num_: 10

  iteration_: 3
  toBeClass_: 8  # the min numbers of points to be one class
  search_c_: 2  # > 1, 2
  intensity_diff_: 3.0
  intensity_cov_: 2.0
  occupancy_: 0.8

  building_: 0  # cluster map
  tree_: 1
  car_: 2

  dynamic_label_: [252, 253, 254, 255, 256, 257, 258, 259]   # refer to semantic scan context

feature:
  kOneThird_: 0.333
  kLinearityMax_: 288.9
  kPlanarityMax_: 288.9
  kScatteringMax_: 2889.9
  kOmnivarianceMax_: 0.278636
  kAnisotropyMax_: 1248.0
  kEigenEntropyMax_: 0.956129
  kChangeOfCurvatureMax_: 0.99702
  kNPointsMax_: 13200.0

  





  



